### PR TITLE
Droidel to standard

### PR DESCRIPTION
**Pull request by ftc**
Added script for conversion of Gradle directory structure. Updated README for usage.

### CODE REVIEW

File: gradle-to-standard.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+#[[ $# -ge 3 ]] || { echo "Usage: $0 [app] [build: eg: debug] [target]" ; exit 1; }
```
Comment (sblackshear) [inline gradle-to-standard.sh]: Delete

File: README.md [hunk 2]
```diff
@@ -42,6 +42,18 @@ In order to run Droidel, you first need to generate a JAR file for the Android f
...
+---------------------------------------------
+The script gradle-to-standard.sh is used to convert the directory structure from an Android gradle project to what Droidel expects.  After the script is built with a command such as:
+
+    ./gradlew assembleDebug 
+   
+The script can be run as 
+
+    sh gradle-to-standard.sh [path to project] [build (eg debug)] [output dir]
+    
+The output direcotry can then be used as a normal project would in droidel.
```
Comment (sblackshear) [inline README.md]: General spelling and grammar check on this section would be good (`direcotry` -> `directory` here, and there's some grammar oddities elsewhere as well).

**Comment by sblackshear** [comment]
Looks good! Could you move the gradle-to-standard.sh script to lib/ and perhaps give it a name like gradle-to-droidel.sh?

**Comment by ftc** [comment]
Sure will do that shortly

**Comment by ftc** [comment]
Ok those should be fixed.

**Comment by sblackshear** [comment]
Thanks!

**Pull request merged**